jobs/github-testers-macros.yaml

```diff
@@ -209,12 +209,11 @@
           #!/bin/bash
           set -xeuo pipefail
 
+          join() { local IFS="$1"; shift; echo "$*"; }
+          RPMS=($RPMS)
           pushd ci-dnf-stack
             pushd rpms
-              echo "$RPMS" | xargs -n 1 curl -L -O
-              if ! ls *.rpm; then
-                exit 1
-              fi
```

Comment:
Why you skipped lines? How you ensure, that all rpms where correctly downloaded?